src/opera/test/pge/dswx_s1/test_dswx_s1_pge.py

```diff
@@ -141,6 +155,41 @@ def _compare_algorithm_parameters_runconfig_to_expected(self, runconfig):
         self.assertEqual(runconfig['processing']['inundated_vegetation']['line_per_block'], 300)
         self.assertEqual(runconfig['processing']['debug_mode'], False)
 
+    def generate_burst_output(self, burst_data, empty_file=False, clear=True):
+        """
+        Add files to the output directory.
+
+        Parameters
+        ----------
+        burst_data: tuple of str
+            Files to add to the output directory.
+        empty_file: bool
+            if 'True' do not add text to the file (leave empty)
+            if 'False' (default) add 'Test data string' to the file
+        clear : bool
+            Clear the output directory before writing new files (default=True)
+
+        """
+        # example of burst data passed to method:
+        # burst_data = ('OPERA_L3_DSWx-S1_burst_1_B01_WTR.tif', 'OPERA_L3_DSWx-S1_burst_1_B02_BWTR.tif',
+        #               'OPERA_L3_DSWx-S1_burst_1_B03_CONF.tif', 'OPERA_L3_DSWx-S1_burst_2_B01_WTR.tif',
+        #               'OPERA_L3_DSWx-S1_burst_2_B02_BWTR.tif', 'OPERA_L3_DSWx-S1_burst_2_B03_CONF.tif')
+
+        # Clears old burst files (default 'True')
+        output_dir = join(self.test_dir, str(self.test_output_dir))
```

Comment:
I don't think you need this `join` here, `self.test_output_dir` should already be set to the path you're looking for.

Reply:
In this case output_dir needs the complete path.  The relative path does not work on the 'os' commands.